go-filecoin-tutorial/Storing-on-Filecoin.md

```diff
@@ -9,23 +9,23 @@ This guide will show you how to store data on Filecoin by making storage deals w
 * [Import data](#import-data)
 * [Propose a storage deal](#propose-a-storage-deal)
 * [Send data and pay miner](#send-data-and-pay-miner)
-* [Retrieve your data](#retrieve-your-data)
+* [Retrieving data](#retrieving-data)
 
 ## How to store on Filecoin
 
-Because Filecoin is a decentralized storage marketplace, storage clients (our term for anyone who wants to store data on the Filecoin network) propose storage deals with storage miners that they feel can meet their storage needs. Once a storage client has proposed a deal, the miner can choose to accept or reject that deal. The client then sends over the data, and the storage deal officially begins!
+Because Filecoin is a decentralized storage marketplace, storage clients (our term for anyone who wants to store data on the Filecoin network) propose storage deals with storage miners that they feel can meet their storage needs. Once a storage client has proposed a deal, the miner can choose to accept or reject that deal. The client then sends over the data, and the storage deal officially begins.
 
-In order to run the remainder of these commands, make sure you have a Filecoin node installed and running. See [Getting Started](Getting-Started) for more information.
+In order to run the remainder of these commands, ensure a Filecoin node is installed and currently running. See the [getting started](Getting-Started) section of this documentation for more information.
 
 ## Test Filecoin with sample data
 
-You can use your own data to test Filecoin. We recommend using small test files while you're still getting used to the different commands.
+Personal data can be used to test Filecoin. It is recommended to use small test files while initially getting used to the different commands.
```

Comment:
Saying personal data here invokes all sorts of privacy concerns. I was going to suggest an alternative phrase, but I think we can deleted the first sentence altogether.

Reply:
Good call, didn't think about that! Shall remove